networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
```

Comment:
Use double quotes `"` for consistency. Not sure whether it has any other consequence though.

Reply:
Acked